r-client/README.md

```diff
@@ -0,0 +1,46 @@
+# Gro API Client R Usage Example
+
+This is a proof of concept use of the Gro API Python client https://github.com/gro-intelligence/api-client, imported and used in R via the `reticulate` R library. No special formatting or configuration needs to be done to the API client itself as plain Python functions interop with R inputs and outputs.
+
+What this Vagrantfile does is it creates an Ubunutu 16.04 virtual machine, installs Python and R dependencies, and runs an R script emulating the quickstart example from https://github.com/gro-intelligence/api-client/blob/development/api/client/samples/quick_start.py outputing an R dataframe at the end.
+
+This is implemented as a Vagrantfile to demonstrate exactly what dependencies are required in this specific test environment. This example is in no way saying you must or should use Vagrant if intending to use the Gro API in R yourself. It is also important to note that because R is not an officially supported language for using the Gro API, it may take extra configuration on the user's part, and not all features are tested.
+
+## Pre-requisites for running this example
+
+* vagrant
+* virtualbox
+
+## Preparation
+
+In [src/testscript.R](src/testscript.R) replace the token string with your own token. See the README in the main repo here https://github.com/gro-intelligence/api-client for details on how to retrieve a token.
```

Comment:
We can link the readme directly as we do for the testscript.R

Reply:
Done